### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by claireetdamien**
We changed something

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,3 +3,4 @@
 de quatchenko
 
 #Were lost
+We are still lost
```
Comment (almereyda) [inline README.md:6]: I don't believe you!

Comment (claireetdamien) [inline README.md:6]: yes we did!

Comment (Cyliane) [inline README.md:6]: OK

Comment (claireetdamien) [inline README.md:6]: Impressive, isn't it?

**Pull request merged**